frontend/tests/components/draft/callout.test.tsx

```
@@ -52,9 +52,9 @@ jest.mock("@lemonsqueezy/wedges", () => ({
 
 describe("CalloutEditor", () => {
   const mockBlock = {
-    __component: "droplets.callout",
+    __component: "droplets.callout" as "droplets.callout",
```

ESLint

[error] Expected a `const` instead of a literal type assertion.

(@typescript-eslint/prefer-as-const)